unit/testing-utils/require_expr.cpp

```diff
@@ -1,108 +0,0 @@
-/*******************************************************************\
-
- Module: Unit test utilities
-
- Author: Diffblue Ltd.
-
-\*******************************************************************/
-
-/// \file
-/// Helper functions for requiring specific expressions
-/// If the expression is of the wrong type, throw a CATCH exception
-/// Also checks associated properties and returns a casted version of the
-/// expression.
-
-#include "require_expr.h"
-
-#include <testing-utils/catch.hpp>
```

Comment:
These are used extensively in unit tests for the TG module. I'm happy to migrate these into TG, however for writing unit tests validating `exprt`s, they are very useful for keeping tests succinct. If we want to expand unit tests in CBMC I suggest keeping them about.

Reply:
Actually I had been looking at a not-up-to-date version of the tree: since  2348d10f7 `require_expr.h` is being used here as well, so we're good.